### PR TITLE
ONNX as one of NNCF main backends

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- ONNX was pulled out from the experimental into the root

### Reason for changes

- ONNX is poised to become one of the server-side components of NNCF as it already contains the basic post-training quantization algorithms.

### Related tickets

76188

### CODE REVIEW

File: nncf/onnx/hardware/pattern_operations.py [hunk 1]
```diff
@@ -13,23 +13,23 @@
...
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXEluMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXPReluMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXSigmoidMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXHardSigmoidMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXHardSwishMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXAddLayerMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXMulLayerMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXDivLayerMetatype
-from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNXSubMetatype
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXConvolutionMetatype
```
Comment (alexsu52) [inline nncf/onnx/hardware/pattern_operations.py:16]: A little sugar:

```
from nncf.onnx.graph.metatypes.onnx_metatypes import (
    ONNXConvolutionMetatype,
    ONNXConvolutionTransposeMetatype,
    ONNXLinearMetatype,
    ...
)
```

**Review by alexsu52 — COMMENTED** [review]
Some comments from my side:
- The name of the `nncf/onnx/common.py` file does not match the contents of the file.
- Do you really need ONNXGraph?

**Comment by andrey-churkin** [comment]
LGTM

**Comment by KodiaqQ** [comment]
> Some comments from my side:
> 
> * The name of the `nncf/onnx/common.py` file does not match the contents of the file.
> * Do you really need ONNXGraph?

`nncf/onnx/common.py` was renamed into `utils.py`.
ONNXGraph uses at least in the `ModelTransformer`. Do you have any suggestions on how to remove or change ONNXGraph?

File: nncf/onnx/hardware/pattern_operations.py (hunk 1 above)
Comment (KodiaqQ) [reply nncf/onnx/hardware/pattern_operations.py]: Thanks, updated.

**Pull request closed without merging**